audb/core/api.py

```diff
@@ -271,7 +271,7 @@ def dependencies(
         try:
             deps = Dependencies()
             deps.load(cached_deps_file)
-        except (AttributeError, FileNotFoundError, ValueError, EOFError):
+        except (AttributeError, EOFError, FileNotFoundError, KeyError, ValueError):
```

Comment:
If this gets thrown only for 3.8 and 3.8 expires in October, how is it warranted that  `KeyError` gets removed in October from the list of exceptions here?

I mean you might get other key errors in the future that might throttle dependency loading a lot by unnecessary dep downloads that go unnoticed. Is there a way to guard against that?

Reply:
True, `KeyError` should should be unreachable in the future. 
I only found it unintuitive to add unreachable code for the 5 months of lifetime remaining for 3.8.

But I shall not be more picky then necessary - I'll go ahead and approve.